src/test/scala/io/iohk/ethereum/blockchain/sync/regular/RegularSyncSpec.scala

```diff
@@ -474,7 +476,7 @@ class RegularSyncSpec
         peersClient.setAutoPilot(new PeersClientAutoPilot)
         override lazy val branchResolution: BranchResolution = stub[BranchResolution]
         (blockchainReader.getBestBlockNumber _).when().returns(0)
-        (branchResolution.resolveBranch _).when(*).returns(NewBetterBranch(Nil)).repeat(10)
+        (branchResolution.resolveBranch _).when(*).returns(NewBetterBranch(Nil)).atLeastOnce()
```

Comment:
not convinced if this is required but the `repeat(10)` seems arbitrary.

Reply:
Indeed, `atLeastOnce` or `anyNumberOf Times` seems more appropriate. I just always wonder why does a method needs to be called so many times. Is it a recursive one?

Reply:
the logic in RegularSync will run a retry on failure. the failure is the missing node (vide a bit lower) which should result in some logic and a retry. the 10 used to be 1 a long time ago.